src/connect.js

```diff
@@ -129,10 +129,10 @@ export default (mapFirebaseToProps = defaultMapFirebaseToProps) => {
       }
 
       render() {
+        const subscriptionProps = this.state.subscriptionsState
         const firebaseProps = mapFirebase(this.props, this.ref, this.firebaseApp)
         const actionProps = pickBy(firebaseProps, isFunction)
-        const subscriptionProps = this.state.subscriptionsState
-        const props = mergeProps(actionProps, subscriptionProps, this.props)
+        const props = mergeProps(this.props, subscriptionProps, actionProps)
```

Comment:
If you'd want to add defaults to `subscriptionProps` as in the example above you would have to filter out `actionProps` by checking if they are functions or values.